apps/frontend/src/libs/constants/dropdown-menu-items.constnant.ts

```diff
@@ -0,0 +1,9 @@
+import { AppRoute } from "~/libs/enums/enums.js";
+
+import { type DropdownMenuItem } from "../types/types.js";
+
+const DROPDOWN_MENU_ITEMS: DropdownMenuItem[] = [
+	{ href: AppRoute.PROFILE, label: "Go to Profile" },
```

Comment:
let's just name it profile as in sidebar
```suggestion
	{ href: AppRoute.PROFILE, label: "Profile" },
```